### PR TITLE
[l] render resource of data type geojson

### PR DESCRIPTION
**Pull request by subhankarb**
We want to render geojson data as a simple map rather than a table.

  * Modified fetchDataPackageAndData to handle geojson type data
    * Have to get data by fetch as datapackage-test gets table data only
  * Added LeafletMap component render in index.jsx
  * Added leaflet css in index.html
  * Added leaflet dependency
  * Added fixer data for geojson

Ref - #122

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -37,8 +38,14 @@ function renderComponentInElement(el) {
...
     let resource = viewutils.findResourceByNameOrIndex(dp, idx)
-    ReactDOM.render(<HandsOnTable resource={resource} idx={idx} />, el);
+    if (resource.format === 'geojson') {
+      ReactDOM.render(<LeafletMap featureCollection={resource._values} idx={idx} />, el)
+    } else {
+      ReactDOM.render(<HandsOnTable resource={resource} idx={idx} />, el)
+    }
   } else if (el.dataset.type === 'data-views') {
-    ReactDOM.render(<MultiViews dataPackage={dp} />, el);
+    if (dp.views !== undefined) {
```
Comment (anuveyatsu) [inline src/index.jsx]: I have fixed it in MultiViews.jsx - when there is no views given, it renders empty div element.

Comment (anuveyatsu) [reply src/index.jsx]: Which one should we keep?

**Review by anuveyatsu — APPROVED** [review]
Generally looks OK

File: src/index.jsx (hunk 1 above)
Comment (subhankarb) [reply src/index.jsx]: I am changing this one

**Pull request merged**